aircraft/breguet_range.py

```diff
@@ -13,34 +12,40 @@ class BreguetRange(Model):
     """
     def setup(self):
 
-        TSFC_min = Var('TSFC_{min}', 0.307, "lb/lbf/hr", "Minimum TSFC")
-        MTOW = Var('MTOW', 10000, "lbf", "Max take off weight")
-        W_e = Var('W_{e}', 7000, "lbf", "Operating Empty Weight")
-        LoverD_max = Var('LoverD_{max}', 15, "-", "Maximum Lift to Drag Ratio")
-        V_max = Var('V_{max}', 420, "knots", "Maximum Velocity")
-
-        #Constants
-        g = Var('g', 9.81, "m/s^2", "gravity")
-
-        #Free Variables
-        R = Var('R', "nautical_miles", "range")
-        V = Var('V', "knots", "Velocity")
-        LoverD = Var('LoverD', "-", "life to drag ratio")
-        TSFC = Var('TSFC', "lb/lbf/hr", "thrust specific fuel consuption")
-        W_init = Var('W_{init}', "lbf", "initial weight")
-        W_fuel = Var('W_{fuel}', "lbf", "fuel weight")
-        z_bre = Var('z_{bre}', "-", "Breguet parameter")
-        t = Var('t', "hr", "time")
-
-        # Set up Model Equations
+        # Fixed Parameters
+        LD_max = Variable('\\left(\\frac{L}{D}\\right)_{max}', 15, '-',
+                     'Maximum lift-to-drag ratio')
+        MTOW   = Variable('MTOW', 10000, 'lbf', 'Max takeoff weight')
+        TSFC   = Variable('TSFC', 0.307, 'lb/lbf/hr',
+                     'Thrust specific fuel consumption')
+        V_max  = Variable('V_{max}', 420, 'knots', 'Maximum velocity')
+        W_e    = Variable('W_{e}', 7000, 'lbf', 'Operating empty weight')
+
+        # Constants
+        g      = Variable('g', 9.81, 'm/s^2', 'Gravitational acceleration')
+
+        # Free Variables
+        LD     = Variable('\\frac{L}{D}', '-', 'Lift-to-drag ratio')
+        R      = Variable('R', 'nautical_miles', 'Range')
+        V      = Variable('V', 'knots', 'Velocity')
+        W_fuel = Variable('W_{fuel}', 'lbf', 'Fuel weight')
+        W_init = Variable('W_{init}', 'lbf', 'Initial gross weight')
+        z_bre  = Variable('z_{bre}', '-', 'Breguet parameter')
```

Comment:
fyi I'm not sure we're agreed on single vs double quotes. I don't care at all. @bqpd seems to use double. just mentioning it since I see you changed all of them -- again I don't care.